nobots.module

```diff
@@ -8,7 +8,9 @@
  * Implements hook_init().
  */
 function nobots_init() {
-  drupal_add_http_header("X-Robots-Tag", "noindex,nofollow,noarchive");
+  if (variable_get('nobots') == TRUE) {
```

Comment:
This works fine but could also be written as `if (variable_get('nobots', TRUE)) {`. Not sure which one would be better but you might want to provide the default with the var_get.